CHANGELOG.md

```diff
@@ -1,6 +1,68 @@
+- Used hashing algorithm is `Blake2b`,
+- We support construction of accounts from both `Secp256k1` (`Base58`) extended public keys (prefixed with `xpub`) or from `Ed25519` (`Base58`) extended public keys (prefixed with `edpk`).
+#### Wallet logic: 
+- Originated accounts are considered as sub-accounts of implicit accounts that originated them,
+#### Explorer:
+- The explorer used by the default (for the moment) is Tzscan (used endpoint: https://api6.tzscan.io/v3),
+- **TODO**: we still miss some endpoints or could improve some others: 
+	- Missing endpoints to : 
+		- Get storage limit,
+		- Get gas limit, 
+		- Get counter,
+		- Get and push raw transactions,
+	- Improvements:
+		- Improve call to get txs related to an address: can have an exposed call to get all types of operations at once, for the moment we have to specify type as parameter for the `GET` request,
+		- TBD: have a call to query operations from list of addresses (could be useful to sync implicit accounts and originated accounts).
+#### Synchronizer:
+- Synchronization of originated accounts comes right after the originator's account is synced, 
+#### Observer:
+- **TODO**: Implement WS from backend side,
+#### Transactions:
+- For the moment we only support `Transaction`, `Reveal`, `Origination` and `Delegation` operations, those are the ones relevant for us,
+ - Parsing and serialization of transaction is based on Zarith encoder/decoder (for more details refer to: http://tezos.gitlab.io/master/api/p2p.html#n-t 
+ implementation is located here : `core/src/bytes/zarith`),
+ - In Tezos, it is possible to have multiple operations in one transaction (e.g. `Reveal` + `Origination` or `Reveal` + `Transaction` ...), 
+ since each operation has it's own properties (fees, gas limit, storage limit ...) that user should set, so we probably won't handle multi-operation transactions.
+ This will be discussed with Libcore's users (e.g. Live team) to see if it is relevant to support multi-operation transactions.
+ - **TODO**: 
+ 	- Handle parsing/serialization of signed transactions,
+ 	- Handle remaining types of operations,
+ 	- TBD: handle multi-operation transaction,
+#### Delegation:	
+- To delegate simply build a `Delegation` transaction with address to which delegate as receiver.  
+#### Parameters:
+- BIP44 Coin type : `1729`,
+- Used address prefixes: 
+	- Implicit (prefix `tz1`): `{0x06, 0xA1, 0x9F}`,
+	- Originated (prefix `KT1`): `{0x02, 0x5A, 0x79}`,
+- Default explorer's endpoint: https://api6.tzscan.io/v3
+#### Tests:
+- Added tests:
+	- Address derivation: `core/test/tezos`,
+	- Zarith encoder: `core/test/bytes/zarith.cpp`,
+	- Account creation and synchronization: `core/test/integration/synchronization/tezos_synchronization.cpp`,
+	- Transaction construction: `core/test/integration/transactions/tezos_transaction_tests.cpp`,
+- TODO: 
+	- Still need to test:
+		- Broadcast of signed transactions,
+		- Test E2E, 
+#### References:
```

Comment:
Please remove this paragraph. It has nothing to do in a changelog either. You can move it to the codebase, for instance, which is a place where people will want to get some references, but not in a changelog.